apis/UserSettings/IUserSettings.h

```diff
@@ -331,6 +332,66 @@ struct EXTERNAL IUserSettings : virtual public Core::IUnknown
 
 };
 
+/**
+ * If a setting was owned by an other component prior to being migrated into UserSettings,
+ * we require the ability to detect when this migration has completed.
+ * The component which was the previous owner of a given setting has the responsibility
+ * to always set the setting on UserSettings interface (During migration AND during first time install).
+ * Until this is done the setting is not considered valid, meaning the the MigrationState will contain requiresMigration=true
+ * All settings not requiring migration would immediately be valid.
+ */
+
+/* @json 1.0.0 @text:keep */
+struct EXTERNAL IUserSettingsInspector : virtual public Core::IUnknown
+{
+    enum { ID = ID_USER_SETTINGS_INSPECTOR };
+
+    ~IUserSettingsInspector() override = default;
+
+    enum SettingsKey : uint32_t
+    {
+        PREFERRED_AUDIO_LANGUAGES = 1           /* @text PREFERRED_AUDIO_LANGUAGES */ ,
```

Comment:
Do we really need @text for each enum value as they are essentially same as the actual name. If not, please remove.